ui/page/privacy/setup_privacy_page.go

```diff
@@ -59,72 +51,86 @@ func NewSetupPrivacyPage(l *load.Load, wallet *dcr.Asset) *SetupPrivacyPage {
 // may be used to initialize page features that are only relevant when
 // the page is displayed.
 // Part of the load.Page interface.
-func (pg *SetupPrivacyPage) OnNavigatedTo() {
-	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
-}
+func (pg *SetupPrivacyPage) OnNavigatedTo() {}
 
 // Layout draws the page UI components into the provided layout context
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
-func (pg *SetupPrivacyPage) Layout(gtx layout.Context) layout.Dimensions {
-	return cryptomaterial.UniformPadding(gtx, pg.privacyIntroLayout)
+func (pg *SetupPrivacyPage) Layout(gtx C) D {
+	return pg.privacyIntroLayout(gtx)
```

Comment:
Please move code on `privacyIntroLayout` function to here, It doesn't make much sense when we have 2 functions that do the same thing

Reply:
Ok.